_data/glossary/annotation.yaml

```diff
@@ -0,0 +1,9 @@
+id: annotation
+name: Annotation
+full-link: docs/concepts/overview/working-with-objects/annotations
+tags:
+- fundamental
+short-description: >
+  An annotation is a key-value pair and used to attach arbitrary non-identifying metadata to objects.
+long-description: >
+  The metadata in an annotation can be small or large, structured or unstructured, and can include characters not permitted by labels. Clients such as tools and libraries can retrieve this metadata.
```

Comment:
Might be useful to link to glossary term for label? (I'll try to get that one merged so that a link doesn't break the build -- if your build does break, that's probably why, and you can feel free to ping me on Slack about it.)

Reply:
Yup, absolutely. I actually added and then removed considering labels weren't created. But now I know someone else has created PR for labels so I will be editing it here. Thanks!